prov/efa/configure.m4

```diff
@@ -52,7 +52,7 @@ AC_DEFUN([FI_EFA_CONFIGURE],[
 				[efa_happy=1],
 				[
 					efa_happy=0
-					AC_MSG_WARN([The EFA provider requires rdma-core v27 or newer.])
+					AC_MSG_WARN([The EFA provider requires rdma-core v31 or newer.])
```

Comment:
According our earlier discussion, I thought we should check at least one of the CQ APIs in your PR before we prompt such version requirement?

Reply:
Ok sure I will add that.

Reply:
Unfortunately `libefa` does not export the wide completion APIs directly. IIUC it's nontrivial to verify the API in this way. The available functions in `libefa` are not sufficient to tell if wide completion is supported by EFA or not.
```
[ec2-user@ip-10-0-15-237 workplace]$ nm -D /usr/lib64/libefa.so
                 U calloc
                 w __cxa_finalize
0000000000000000 A EFA_1.0
0000000000000000 A EFA_1.1
0000000000004570 T efadv_create_driver_qp
0000000000004730 T efadv_create_qp_ex
0000000000005490 T efadv_query_ah
0000000000003b80 T efadv_query_device
                 U __errno_location
                 U free
                 w __gmon_start__
                 U ibv_cmd_alloc_pd
                 U ibv_cmd_create_ah
                 U ibv_cmd_create_cq_ex <--- this is declared in libibverbs, not efa
                 U ibv_cmd_create_qp_ex
                 U ibv_cmd_dealloc_pd
                 U ibv_cmd_dereg_mr
                 U ibv_cmd_destroy_ah
                 U ibv_cmd_destroy_cq
                 U ibv_cmd_destroy_qp
                 U ibv_cmd_get_context
                 U ibv_cmd_modify_qp
                 U ibv_cmd_query_device_any
                 U ibv_cmd_query_port
                 U ibv_cmd_query_qp
                 U ibv_cmd_reg_mr
                 w _ITM_deregisterTMCloneTable
                 w _ITM_registerTMCloneTable
                 U malloc
                 U memcpy
                 U memset
                 U mmap
                 U munmap
                 U pthread_spin_destroy
                 U pthread_spin_init
                 U pthread_spin_lock
                 U pthread_spin_unlock
                 U __snprintf_chk
                 U __stack_chk_fail
                 U sysconf
                 U _verbs_init_and_alloc_context
                 U __verbs_log
                 U verbs_register_driver_34
                 U verbs_set_ops
                 U verbs_uninit_context
```